pkgs/development/ruby-modules/bundler-env/default.nix

```diff
@@ -46,11 +46,20 @@ let
 
   importedGemset = import gemset';
 
-  filteredGemset = (lib.filterAttrs (name: attrs:
-    if (builtins.hasAttr "groups" attrs)
-    then (builtins.any (gemGroup: builtins.any (group: group == gemGroup) groups) attrs.groups)
-    else true
-  ) importedGemset);
+  platformMatches = attrs: (
+  !(attrs ? "platforms") ||
+    builtins.any (platform:
+      platform.engine == ruby.rubyEngine &&
+        (!(platform ? "version") || platform.version == ruby.version.majMin)
```

Comment:
Certainly there'll need to be a mapping from Gemfile platform version to NixOS ruby.version.majMin. I'm not sure the two should be identical. At the moment, Bundix doesn't yet include this information in the gemset.nix (although that's what I'm up to now...) so it's not pressing.

I wonder if the best solution isn't to provide a mapping function in bunder-env itself?